app/packages/vhx-js-mithril/docs/guide.html

```diff
@@ -0,0 +1,118 @@
+      <p class="margin-bottom-medium text-2">Be sure to make the files <code class="text--yellow">ui</code>, <code class="text--yellow">state</code>, <code class="text--yellow">controller</code>, and <code class="text--yellow">constants</code> for each component/layout. See the next sections for definitions of what each of the parts of a component are.</p>
+    </div>
+  </section>
+</template>
+
+<template name="guide__js_mithril_order">
+  <section id="guide--js-mithril-order">
+    <!-- File Order -->
+    <div class="margin-vert-large">
+      <h4 class="margin-bottom-medium padding-bottom-small border-bottom border--gray-light text--bold text--gray">2. File Order</h4>
+      <p class="margin-bottom-medium text-2">When setting up your application manifest, be sure to include your app scripts in the order shown to ensure proper initialization.</p>
+    </div>
+  </section>
+</template>
+
+<template name="guide__js_mithril_namespace">
+  <section id="guide--js-mithril-namespace">
+    <!-- Namespace -->
+    <div class="margin-vert-large">
+      <h4 class="margin-bottom-medium padding-bottom-small border-bottom border--gray-light text--bold text--gray">3. Namespace ( <code class="text--yellow">scope.js</code> )</h4>
+      <p class="margin-bottom-medium text-2">To both avoid polluting the global namespace and for providing an organized way to keep our Mithril code in one place we use the <code class="text--yellow">vhxm</code> namespace. Within the <code class="text--yellow">vhxm</code> object all components, layouts, and models live.</p>
+      <p class="margin-bottom-medium text-2">The <code class="text--yellow">vhxm</code> object should be setup in a <code class="text--yellow">scope.js</code> file. Here you can establish a broad overview/map by definining the broad stroke components of your application.</p>
+    </div>
+  </section>
+</template>
+
+<template name="guide__js_mithril_layouts">
+  <section id="guide--js-mithril-layouts">
+    <!-- Layouts -->
+    <div class="margin-vert-large">
+      <h4 class="margin-bottom-medium padding-bottom-small border-bottom border--gray-light text--bold text--gray">4. Layouts</h4>
+      <p class="margin-bottom-medium text-2">Layouts are just Mithril Components that we use as our top level components, initiated in our routes and that contain all other components.</p>
+      <p class="margin-bottom-medium text-2">They aren't any different than other components in their structure, but we name and put them under the layouts object for organizational purposes.</p>
+      <p class="margin-bottom-medium text-2">In Mithril, a component is an object that has at the very least a view method that returns a Mithril Virtual DOM tree (via <code class="text--yellow">m()</code>). Components are the building blocks of a Mithril application. You can include sub components in a component by using Mithril's <code class="text--yellow">m.component()</code> method.</p>
```

Comment:
Should this be moved into an earlier section, establishing Mithril fundamentals? Maybe into the Intro, since components are a fundamental element of Mithril's work.